code/modules/mob/living/carbon/alien/emote.dm

```diff
@@ -0,0 +1,58 @@
+/datum/emote/alien/scretch
+	key = "scretch"
+	key_third_person = "scretches"
+	message = "scretches"
+	emote_type = EMOTE_AUDIBLE
+
+/datum/emote/alien/tail
+	key = "tail"
+	key_third_person = "tails"
+	message = "waves their tail"
+
+/datum/emote/alien/sign
+	key = "sign"
+	key_third_person = "signs"
+	message = "signs"
+	message_param = "signs %t"
+
+/datum/emote/alien/sign/select_param(mob/user, params, msg)
+	if(text2num(params))
+		return "[msg] the number [params]"
+	return msg
+
+/datum/emote/alien/sound
```

Comment:
You wouldn't need to have a subtype for this if there wasn't the ridiculous "MUST BE AUDIBLE EMOTE TO HAVE SOUND PLAYED" requirement.